js/views/errorScreen.js

```diff
@@ -13,9 +13,23 @@ var ErrorScreen = React.createClass({
   render: function() {
     var errorTitle, errorDescription, errorAction;
     if (CONSTANTS.ERROR_MESSAGE.hasOwnProperty(this.props.errorCode.code)){
+
+      var errorAction = CONSTANTS.SKIN_TEXT.ERROR_ACTION;
+      if( CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].action ){
+        errorAction = CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].action;
+      }
+
       errorTitle = Utils.getLocalizedString(this.props.language, CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].title, this.props.localizableStrings)
       errorDescription = Utils.getLocalizedString(this.props.language, CONSTANTS.ERROR_MESSAGE[this.props.errorCode.code].description, this.props.localizableStrings);
-      errorAction = Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.ERROR_ACTION, this.props.localizableStrings);
+      /// TODO - need to make countdown functionality display for all languages
+      var startTime = this.props.errorCode.flight_start_time;
+      if (this.props.errorCode.code === CONSTANTS.ERROR_CODE.FUTURE 
+          && this.props.language === CONSTANTS.LANGUAGE.ENGLISH 
+          && startTime != null && !isNaN(startTime)){
+        errorDescription = "This video will be available in " + Utils.getStartCountdown(startTime * 1000 - Date.now());
+      }
+      errorAction = Utils.getLocalizedString(this.props.language, errorAction, this.props.localizableStrings);
+      console.log("action: " +errorAction);
```

Comment:
Not super critical, but would suggest removing this or switching it to `OO.log`

Reply:
@jpfranco actually this should have been removed, was for dev testing only